firstvoices/backend/views/storypage_views.py

```diff
@@ -114,25 +112,7 @@ def get_queryset(self):
             .order_by("ordering")
             .all()
             .prefetch_related(
-                Prefetch(
```

Comment:
I think we could leave this here as well, right? Have the APIs return the audio in the right order any time they are returning story page data?

Reply:
Oh, I get it now ha ha. Sorry I didn't realize this was a new media prefetch function just for the ordered data! Looks good.